src/components/Modal.js

```diff
@@ -204,7 +203,6 @@ class Modal extends React.Component {
               ref={ref => this._modalContent = ref}
               role={this.props.role}
               style={Object.assign({}, styles.content, this.props.contentStyle)}
-              tabIndex={0}
```

Comment:
Just to be safe. You didn't run into any console errors around assigning undefined to the role here on line 204 above now that the default has been removed?

Reply:
Double checked with passing no role prop. No console errors